storage/innobase/xtrabackup/src/redo_log_consumer.h

```diff
@@ -0,0 +1,51 @@
+/******************************************************
+Copyright (c) 2019,2022 Percona LLC and/or its affiliates.
```

Comment:
2022 is sufficient (and all other places when new file is created)

Reply:
@altmannmarcelo

Reply:
@satya-bodapati Done